docs/it/percorsi-di-formazione-alternativi.md

```diff
@@ -0,0 +1,165 @@
+
+In poche parole: non basta il tempo "vivo" proposto dal percorso di formazione, questo perchè è necessario che chi sta imparando riesca ad "entrare" nell'ottica... ovvero acquisire il **mindset** da developer.
+
+"Ma se ho un lavoro come faccio ad investire tutto questo tempo?"
+> Generalmente se ci si affida ad un percorso di formazione per imparare a programmare sarebbe meglio dedicarsi fulltime ad esso, perchè la mole di nozioni da acquisire nel pochissimo tempo a disposizione (tre o sei mesi) è molto elevata.
+> Sarebbe più consigliabile un altro tipo di approccio per chi non dispone di così tanto tempo: studiare da autodidatta oppure attraverso un percorso personalizzato; in questo caso però non sarà possibile ottenere un risultato in soli tre/sei mesi ma ci vorrà molto più tempo.
+
+Quindi se non avete pazienza, tempo, passione, curiosità e voglia è meglio lasciare perdere.
+
+## Terza fase: gli sbocchi lavorativi
+
+Gli sbocchi lavorativi nel settore della programmazione sono tanti.
+La programmazione è un trend in costante crescita dato che è una materia in continua evoluzione.
+
+Scommettere quindi su una tecnologia piuttosto che su un'altra per impostare la propria carriera lavorativa potrebbe essere un errore: il rischio di diventare un "dinosauro" e ritrovarsi fuori mercato è troppo alto.
+
+Se si vuole rimanere appetibili nei confronti delle aziende e del mercato dobbiamo quindi essere in grado di adattarci ai cambiamenti e, soprattutto, aggiornarci (eh si... spesso questo processo avviene in completa autonomia).
+
+Facciamo un esempio, rimanendo sempre nel campo dei siti web.
+Prima i Layout (la parte grafica del sito web) non seguivano degli "standard" veri e propri, poi ad un certo punto sono nati dei _Framework_ (si tratta di un insieme di strumenti) che hanno definito tali standard (come ad esempio il concetto di colonne molto utile per rendere un sito web _responsive_ ovvero adattabile ad ogni dispositivo).
+
+Quindi chi progettava i Layout ha dovuto apprendere tali "standard" per non rimanere fuori mercato dato che, ovviamente, tutti i siti ad oggi sono navigabili e fruibili anche dal nostro Smartphone.
+La figura _Web Designer_ di ieri, che si limitava solamente ad impostare graficamente gli elementi a schermo, è diventata oggi _Frontend Developer_ che è in grado di mettere mano a _Framework_ e varie _librerie_.
+
+Questo concetto spiega bene 2 cose:
+
+- bisogna continuare ad imparare e ad aggiornarsi
+- non si può fare affidamento su di una tecnologia
+
+"Tutto bellissimo, ma come faccio a sapere se un percorso di formazione mi permette di lavorare dopo?
+Chi mi dà questa certezza dato che spesso e volentieri hanno dei costi alti?"
+Il settore della programmazione, come detto in precedenza, non è un settore in perdita e non è un settore saturo.
```

Comment:
```suggestion
Il settore della programmazione, come dimostrano le statistiche, non è un settore in perdita e non è un settore saturo, perlomeno nel periodo in cui questo contenuto è stato scritto.
```

Ti suona meglio @nicolaerario ?

Reply:
Benissimo. Ho riletto per trovare dove era stato detto in precedenza; mi sentivo a disagio. Grazie